src/main/java/hudson/plugins/jira/JiraBuildAction.java

```diff
@@ -11,11 +15,12 @@
  * @author Kohsuke Kawaguchi
  */
 public class JiraBuildAction implements Action {
-    public final AbstractBuild<?, ?> owner;
+    
+	public final Run<?, ?> owner;
```

Comment:
I don't think so. Run is super class of AbstractBuild so no java compatibility can be broken. If you think compatibility with serialization of jobs and runs, i read in hudson.model.Action: Actions are often persisted as a part of {@link Actionable} (for example with {@link Build}) via XStream. If actions are persited via XStream, it's not problem as i known XStream. I add junit test in next commit to this PR that proof it.